adagrams/game.py

```diff
@@ -1,11 +1,96 @@
+import random
+
+
+LETTER_POOL = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D',
+        'D','D','D','E','E','E','E','E','E','E','E','E','E','E','E','F',
+        'F','G','G','G','H','H','I','I','I','I','I','I','I','I','I','J',
+        'K','L','L','L','L','M','M','N','N','N','N','N','N','O','O','O',
+        'O','O','O','O','O','P','P','Q','R','R','R','R','R','R','S','S',
+        'S','S','T','T','T','T','T','T','U','U','U','U','V','V','W','W',
+        'X','Y','Y','Z']
+
+
 def draw_letters():
-    pass
+    working_letter_pool = LETTER_POOL.copy()
+    hand = []
+
+    for index in range(10):
+        drawn_letter = random.choice(working_letter_pool)
+        if drawn_letter in working_letter_pool:
+            hand.append(drawn_letter)
+        working_letter_pool.remove(drawn_letter)  
```

Comment:
This is an optimal solution, outstanding 💫 however, I think this could work without the conditional. If you think about it, why are we checking if the `drawn_letter` is in the `working_letter_pool` if we randomly choice a letter from there? I'd like to offer this refactor:
```python
    for index in range(10):
        drawn_letter = random.choice(working_letter_pool)
        hand.append(drawn_letter)
        working_letter_pool.remove(drawn_letter)  
    return hand
```
Just something to consider 😁